tests/openvino/test_diffusion.py

```diff
@@ -379,15 +377,13 @@ def test_height_width_properties(self, model_arch: str):
         self.assertEqual(ov_pipeline.height, height)
         self.assertEqual(ov_pipeline.width, width)
 
-    @pytest.mark.run_slow
-    @slow
     @require_diffusers
     def test_textual_inversion(self):
         # for now we only test for stable-diffusion
         # this is very slow and costly to run right now
 
-        model_id = "runwayml/stable-diffusion-v1-5"
```

Comment:
I guess the comment "this is very slow and costly to run right now" can be removed now

Reply:
removed